src/compute/src/render/mod.rs

```diff
@@ -214,6 +214,7 @@ pub fn build_compute_dataflow<A: Allocate>(
                         inner.clone(),
                         format!("flow_control_input({source_id})"),
                         probe.clone(),
+                        Timestamp::minimum(),
```

Comment:
The summary type is not necessarily the same as the timestamp type so if you want to be generic here `Default::default()` is a better choice